homeassistant/components/rest/binary_sensor.py

```diff
@@ -44,6 +47,7 @@
             [HTTP_BASIC_AUTHENTICATION, HTTP_DIGEST_AUTHENTICATION]
         ),
         vol.Optional(CONF_HEADERS): {cv.string: cv.string},
+        vol.Optional(CONF_JSON_ATTRS): vol.Any(cv.template_complex, cv.template),
```

Comment:
We should call it `attributes` and it should accept a `{str: cv.template_complex}`